nifi-commons/nifi-web-client-api/src/main/java/org/apache/nifi/web/client/api/MultipartFormDataStreamBuilder.java

```diff
@@ -55,4 +55,12 @@ public interface MultipartFormDataStreamBuilder {
      * @return Builder
      */
     MultipartFormDataStreamBuilder addPart(String name, HttpContentType httpContentType, byte[] bytes);
+
+    default MultipartFormDataStreamBuilder addPart(String name, String fileName, HttpContentType httpContentType, InputStream inputStream) {
+        return addPart(name, httpContentType, inputStream);
+    }
+
+    default MultipartFormDataStreamBuilder addPart(String name, String fileName, HttpContentType httpContentType, byte[] bytes) {
+        return addPart(name, httpContentType, bytes);
+    }
```

Comment:
For completeness, it would be helpful to add JavaDoc comments like the other methods.